ember/app/components/project-detailed/component.js

```diff
@@ -28,4 +30,19 @@ export default class ProjectDetailedComponent extends Component {
     }
     this.store.findRecord('project', this.args.project.id);
   });
+
+  deleteProject = dropTask(async () => {
+    try {
+      if (
+        !(await confirm(`Confirm deleting project ${this.args.project.name}`))
```

Comment:
Let's make this a little bit less technical. Usually a user would expect something like: "Are you sure you want to delete the project XY? This action can't be undone."

Reply:
done here: https://github.com/adfinis/Outdated/pull/71/commits/33d46ec7358b6d282d53fb3f598d712751354b04